Lib/modulefinder.py

```diff
@@ -69,16 +138,16 @@ def __repr__(self):
 
 class ModuleFinder:
 
-    def __init__(self, path=None, debug=0, excludes=[], replace_paths=[]):
+    def __init__(self, path=None, debug=0, excludes=None, replace_paths=None):
```

Comment:
While this seems like a good change to me, it also seems worth mentioning in one of the NEWS entries (probably by adding it to https://github.com/python/cpython/pull/11787/files#diff-0c343f3267eee48d7cb592115947061b ).